crates/api/src/models/responses.rs

```diff
@@ -9,3 +10,13 @@ pub struct GetGreetingsResponse {
 pub struct CreateGreetingResponse {
     pub greeting: Greeting,
 }
+
+#[derive(serde::Serialize, serde::Deserialize)]
+pub struct LoginResponse {
+    pub token: String,
+}
+
+#[derive(serde::Serialize, serde::Deserialize)]
+pub struct CreateUserResponse {
+    pub user: User,
```

Comment:
I don't see values on returning user object here. What about returning token? As a user just signed up, we can treat them as logged in too.